src/collection-browser.ts

```diff
@@ -1700,11 +1700,30 @@ export class CollectionBrowser
     infinite-scroller.grid {
       --infiniteScrollerCellMinWidth: var(
         --collectionBrowserCellMinWidth,
-        18rem
+        17rem
       );
       --infiniteScrollerCellMaxWidth: var(--collectionBrowserCellMaxWidth, 1fr);
     }
 
+    /* Allow tiles to shrink a bit further at smaller viewport widths */
+    @media screen and (max-width: 880px) {
```

Comment:
mixing a couple of my favorite css tricks: media queries with global vars! :nice: